pyiceberg/table/update/__init__.py

```diff
@@ -90,7 +90,13 @@ class AddSchemaUpdate(IcebergBaseModel):
     # This field is required: https://github.com/apache/iceberg/pull/7445
     last_column_id: int = Field(alias="last-column-id")
 
-    initial_change: bool = Field(default=False, exclude=True)
+    initial_change: bool = Field(
```

Comment:
It is clean to go through the validation cycle, but I would be surprised if anyone would be relying on these properties